<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dokument Export</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Marker</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbSeperatePages">
        <property name="text">
         <string>as single page</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbMergeAsDocument">
        <property name="text">
         <string>arrange as new document</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbCompletePage">
        <property name="text">
         <string>Insert pagebreak, if the page would be uncomplete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Other options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Zoom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="dsZoom">
        <property name="maximum">
         <double>5.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Exportformat</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>478</width>
        <height>339</height>
       </rect>
      </property>
      <attribute name="label">
       <string>LaTeX Sourceode</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Path</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="rbAbsolutePaths">
            <property name="text">
             <string>absolute</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbFilenameOnly">
            <property name="text">
             <string>only filename</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Filepath</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePath">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnChoosePath">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbWholeDocument">
         <property name="text">
          <string>Complete document</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCreateOutput">
         <property name="text">
          <string>Create sourcecode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="txtOutput"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>492</width>
        <height>337</height>
       </rect>
      </property>
      <attribute name="label">
       <string>PDF</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QPushButton" name="pbSavePdf">
         <property name="text">
          <string>Save PDF</string>
         </property>
         <property name="icon">
          <iconset theme="document-save-as"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Statusmessages</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lwStatus"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
